fedgov-cv-map-services-proxy/src/main/java/gov/usdot/cv/esrimap/models/EsriSpatialReference.java

```diff
@@ -21,25 +21,35 @@ public class EsriSpatialReference {
     @JsonProperty("wkid")
     @JsonAlias("wkid")
     private double wkid;
+    private double vcsWkid;
```

Comment:
In case of the variation in the JSON fields, we could add `@JsonIgnoreProperties(ignoreUnknown = true)` at the top of `public class EsriSpatialReference {` definition to ignore any unknown properties.